Style-Guide.md

```diff
@@ -43,6 +43,9 @@ Please don't use the following name for variable since it is used by our `entryp
 | planet-db-host 	| used in the docker script 	|
 | planet-db-port 	| used in the docker script 	|
 
+### il8n
```

Comment:
@gcantieni can we add few more examples as people might not be familiar with interpolation or some other terms. Also i18n can be used for attributes such as `i18n-title` `i18n-placeholder`